### PR TITLE
Enable artwork

### LINKED ISSUE #62: Podcasts - enable artwork (photo)
- Cloudinary setup
- Enable active storage
- Add the photo field to the db

See the tutorial for this...

### PR DESCRIPTION
**Pull request by b-lauren**
Closes #62 

Enabled artwork

### CODE REVIEW

**Review by solowresidual — APPROVED** [review]
Looks good nice work! Does Episode need to be able to hand a photo, too?

**Comment by b-lauren** [comment]
@solowresidual episode image will just be the podcast artwork, right? Can add it though!

**Review by johnjoubert — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -36,6 +36,8 @@ yarn-debug.log*
 .yarn-integrity
 # Ignore .env file containing credentials.
 .env*
+echo '.env*'
 # Ignore Mac and Linux file system files
 *.swp
 .DS_Store
+.env*
```
Comment (johnjoubert) [inline .gitignore:43]: We can also remove this one actually because it's already on line 38.
```suggestion
```

**Comment by b-lauren** [comment]
@johnjoubert sorted!

**Review by johnjoubert — APPROVED** [review]

**Review by solowresidual — APPROVED** [review]

**Pull request merged**